server/utils/hashing.ts

```diff
@@ -0,0 +1,37 @@
+import bcrypt from "bcrypt";
+
+const saltRounds: number = 10; // Number of salt rounds for hashing
+
+// Hash a password during user registration
```

Comment:
I personally don't think comments are needed for either `hashPassword` or `isPassowrd`. Their names are already self documenting. However, if you are going to comment the function, use a docstring.